exo-test/src/service-tester.ls

```diff
@@ -42,9 +43,18 @@ class ServiceTester extends EventEmitter
 
 
   _start-dependencies: (done) ~>
-    for dep of @service-config.dependencies
-      DockerHelper.ensure-container-is-running "test-#{dep}", dep
-    wait 500, done
+    dependencies = []
+    for dependency-name, dependency-config of @service-config.dependencies
+      dependencies.push do
+        {
+          container-name: "test-#{dependency-name}"
+          port: '-p 27017:27017'
+          dependency-name: dependency-name
+          online-text: dependency-config.docker_flags?.online_text
+        }
+    async.each-series dependencies, DockerHelper.ensure-container-is-running, (err) ~>
+      | err  => throw new Error err
```

Comment:
Are you sure throwing an error is the right course of action here? We should return it via the `done` function.